### PR TITLE
Issue 91 - Add email plugin

### LINKED ISSUE #91: Add Share by email action
We need to make visible the `@@send_to` view available on Plone:

https://www.conversaafiada.com.br/brasil/golpe-foi-para-abafar-a-lava-jato/sendto_form

### PR DESCRIPTION
**Pull request by rodfersou**
closes #91

### CODE REVIEW

**Comment by rodfersou** [comment]
I don't know what is going on with these RF tests failing

File: sc/social/like/tests/ManagePluginUtils.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from plone import api
+from sc.social.like.interfaces import ISocialLikeSettings
+
+
+def enable_plugin(plugin):
+    """Enable sc.social.like plugins."""
+    record = dict(name='plugins_enabled', interface=ISocialLikeSettings)
+    plugins_enabled = api.portal.get_registry_record(**record)
+    record['value'] = plugins_enabled + (plugin, )
+    api.portal.set_registry_record(**record)
```
Comment (rodfersou) [inline sc/social/like/tests/ManagePluginUtils.py]: @hvelarde any idea on why this  is not working?

Comment (hvelarde) [inline sc/social/like/tests/ManagePluginUtils.py]: probably you need to commit the transaction; let's ask @datakurre.

Comment (rodfersou) [inline sc/social/like/tests/ManagePluginUtils.py]: I already tried to commit the transaction, and didn't work...

Comment (rodfersou) [inline sc/social/like/tests/ManagePluginUtils.py]: my bad.. now it worked locally!

File: sc/social/like/tests/test_plugin_email.robot [hunk 2]
```diff
@@ -38,6 +38,7 @@ Create Document
     Page Should Contain  Changes saved.
 
 Open Workflow Menu
+    Sleep  1s  Wait for contentmenu to load
```
Comment (rodfersou) [inline sc/social/like/tests/test_plugin_email.robot]: This fixed my test locally (I'll do the same in other PR to fix other tests)

**Comment by rodfersou** [comment]
@hvelarde green

**Comment by rodfersou** [comment]
@hvelarde to test if email is sent we need a smtp server AFAIK @@sendto_form don't have any tests already? It worked in one of our customers

**Comment by rodfersou** [comment]
@hvelarde notes: use **Products.PrintingMailHost** to create a test to send the email

**Comment by rodfersou** [comment]
@hvelarde green

File: sc/social/like/browser/images/ico-email.svg [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (hvelarde) [inline sc/social/like/browser/images/ico-email.svg]: from where did you get this file? that's not the one we have talked about, but this one:

https://upload.wikimedia.org/wikipedia/commons/d/df/Aiga_mail.svg

Comment (rodfersou) [inline sc/social/like/browser/images/ico-email.svg]: I updated the issue, yesterday when we fixed the colors @agnogueira suggested to change the icon.

Please keep this file separated from CSS now, we will mix it when use webpack.

**Comment by hvelarde** [comment]
the upgrade step is still missing.

**Pull request merged**